server/routes/temporaryabsences/confirmTemporaryAbsenceAddedToRollController.ts

```diff
@@ -1,9 +1,27 @@
 import { RequestHandler } from 'express'
+import type { PrisonService } from '../../services'
 
 export default class ConfirmTemporaryAbsenceAddedToRollController {
+  public constructor(private readonly prisonService: PrisonService) {}
+
   public view(): RequestHandler {
     return async (req, res) => {
-      return res.render('pages/temporaryabsences/confirmTemporaryAbsenceAddedToRoll.njk', {})
+      const { prisonNumber } = req.params
+      const { activeCaseLoadId } = res.locals.user
+      const { firstName, lastName } = req.flash('prisoner')?.[0] as Record<string, string>
+
+      const prison = await this.prisonService.getPrison(activeCaseLoadId)
+
+      if (!firstName || !lastName || !prisonNumber || !prison) {
```

Comment:
meh.. it's not possible to not have a prisonNumber or a prison - the route wouldn't fire if you don't have a prisonNumber, and the getPrison call would throw an exception..